packages/apps/src/Menu/index.tsx

```diff
@@ -28,16 +28,16 @@ interface Props {
 function createExternals (t: TFunction): ItemRoute[] {
   return [
     {
-      href: 'https://github.com/polkadot-js/apps',
+      href: 'https://github.com/CherryNetwork',
```

Comment:
Shouldnt it be this? https://github.com/CherryNetwork/apps

Reply:
Okay fixing it....